region/maridia/outer/Mt. Everest.json

```diff
@@ -4359,6 +4435,7 @@
       ],
       "resetsObstacles": ["A", "B"],
       "farmCycleDrops": [{"enemy": "Sciser", "count": 3}],
+      "flashSuitChecked": true,
```

Comment:
h_midAirShootUp
The farm strats that use Powamps to move around can't reach them from the ground